packages/venia-concept/src/components/MiniCart/kebab.js

```diff
@@ -19,6 +19,9 @@ class Kebab extends Component {
     componentDidMount() {
         document.addEventListener('click', this.handleDocumentClick);
         document.addEventListener('touchend', this.handleDocumentClick);
+        if (this.props.isOpen) {
+            this.setState({ isOpen: true });
+        }
```

Comment:
Generally, initial state that depends on props [should be set in the constructor](https://reactjs.org/docs/react-component.html#constructor).

```es6
constructor(props) {
  super(props)

  this.state = { 
    isOpen: !!props.isOpen
  }
}
```